modules/installation-custom-aws-vpc.adoc

```diff
@@ -33,6 +33,15 @@ Your VPC must meet the following characteristics:
 
 If you use a cluster with public access, you must create a public and a private subnet for each availability zone that your cluster uses. The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. Review the current link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to ensure that the installation program can add a tag to each subnet that you specify.
 
+If you are working in a disconnected environment, you are unable to reach the
+public IPs for EC2 and ELB endpoints. To resolve this, you must create a VPC
+endpoint and attach it to the subnet that the clusters are using. The endpoints
+should be named as follows:
+
+* `ec2.<region>.amazonaws.com`
```

Comment:
Do we want to specify what to provide for `<region>`?

Reply:
@pweil- @sdodson Should we list options here? Thanks!

Reply:
The list of regions will change over time and I'd expect an AWS admin to be able to extrapolate from what you have already.

Reply:
> The list of regions will change over time and I'd expect an AWS admin to be able to extrapolate from what you have already.

+1.  If we want to do anything I'd just link to Amazon documentation and not try to maintain the canonical list here.